packages/mds-geography-service/repository/index.ts

```diff
@@ -240,6 +241,55 @@ class GeographyReadWriteRepository extends ReadWriteRepository {
     }
   }
 
+  public isPublished = (geography: GeographyDomainModel) => {
+    return Boolean(geography.publish_date)
+  }
+
+  /**
+   * Deletes geographyMetadata by geography_id
+   * @param geography_id
+   * @returns geography_id
+   */
+  public deleteGeographyMetadata = async (geography_id: UUID): Promise<UUID> => {
+    try {
+      const connection = await this.connect('rw')
+      await connection.getRepository(GeographyMetadataEntity).delete({ geography_id })
```

Comment:
Does this let `deleteGeographyMetadata` bypass the `isPublished` check?

Reply:
from what I've read, Metadata is actually mutable, forever, based on the implementations in mds-db. 
Maybe I'm missing some Docs that say otherwise? I just tried to keep to the spirit of the implementation and not re-base any of the feature-tests.

Reply:
@nessur is correct. It's the main reason the metadata exists in the first place.